pages/osx/osascript.md

```diff
@@ -0,0 +1,27 @@
+# osascript
+
+> Run AppleScript or JavaScript for Automation (JXA) from the command line.
+
+- Run an AppleScript command:
+
+`osascript -e '{{say "Hello world"}}'`
+
+- Run multiple AppleScript commands:
+
+`osascript -e '{{say "Hello"}}' -e '{{say "world"}}'`
+
+- Run an AppleScript file:
+
+`osascript {{path/to/apple.scpt}}`
```

Comment:
Does the extension matter ? If its not 'scpt', will it fail ? If yes, then it should probably be outside the token`{{}}`